### PR TITLE
show(): Fixed documentation

### PR DESCRIPTION
**Pull request by eironman**
There was a contradiction between show() and css() documentation about !important behaviour

### CODE REVIEW

File: entries/show.xml [hunk 1]
```diff
@@ -32,10 +32,9 @@
...
-    <p><strong>Note: </strong> If using !important in your styles, such as
-    <code>display: none !important</code>,
-    it is necessary to override the style using <code>.css( "display", "block !important")</code> should you wish for <code>.show()</code> to function correctly.</p>
+    <p>The matched elements will be revealed immediately, with no animation. This is roughly equivalent to calling <code>.css("display", "block")</code>, except that the <code>display</code> property is restored to whatever it was initially. If an element has a <code>display</code> value of <code>inline</code>, then is hidden and shown, it will once again be displayed <code>inline</code>.</p>
+    <p><strong>Note: </strong> If using !important in your styles, such as <code>display: none !important</code>, <code>.show()</code> will not override <code>!important</code>.
+    It is recommended to use different classes with <code>.addClass()</code>, <code>.removeClass()</code> or <code>.toggleClass()</code>. Another approach can be the use of <code>.attr("style","display:block !important")</code>, being careful not to overwrite the style attribute of the element.</p>
```
Comment (AurelioDeRosa) [inline entries/show.xml]: Hi. The PR looks good. The only change required is to have the correct spacing here:

```js
.attr("style","display:block !important")
```

It should be:

```js
.css( "display", "block !important" )
```

Comment (mgol) [inline entries/show.xml]: @AurelioDeRosa Your two code snippets differ not only in spacing. :-)

**Comment by AurelioDeRosa** [comment]
lol

You're right. Wrong copy/paste.

**Comment by eironman** [comment]
Ok, I fixed it, but I did a mess updating the commit comments with amend. Now there are 5 commits for this small change. Is it fine or should I start again?

**Comment by AurelioDeRosa** [comment]
No worries, I'll rebase anyway.

**Comment by mgol** [comment]
I tweaked the text a little: the final sentence was a little misleading as using `.attr()` will overwrite the `style` attribute.

Landed, thanks!

**Pull request closed without merging**